pages/production.md

```diff
@@ -37,8 +37,8 @@ Or when using Gradle:
 
 This will generate two files (if your application is called "jhipster"):
 
-*   `target/jhipster-0.0.1-SNAPSHOT.war`
```

Comment:
we should keep both lines: target and build/libs, as it concerns Maven and Gradle too

Reply:
hmmm you're right indeed